CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # Release History
 
+## 1.19.0 (TBD)
+
+### Snowpark Python API Updates
+
+#### Improvements
+
+### Snowpark pandas API Updates
+
+#### New Features
+
+#### Bug Fixes
+
+#### Improvements
+
+- Expand support for empty DataFrames in `DataFrame.pivot_table`.
```

Comment:
this description here is too vague, can you add more details about the "empty dataframe" case you are talking about? do you mean handle pivot on empty dataframe with no rows case to match native pandas behavior?

Reply:
Yup - I'll go ahead and update this!